src/github_runner_manager/openstack_cloud/openstack_cloud.py

```diff
@@ -268,6 +304,7 @@ def _delete_instance(self, conn: OpenstackConnection, full_name: str) -> None:
         ) as err:
             raise OpenStackError(f"Failed to remove openstack runner {full_name}") from err
 
+    @_catch_openstack_errors
```

Comment:
technically not needed here (and not for `get_server_name` below), but put there to avoid future mistakes (when an engineer adds code with openstack api calls, but forgets to add the decorator)